### ISSUE TITLE
bug:(form-field) icon-button used as suffix/prefix displays wrong on Edge

### ISSUE THREAD

**Issue by IlCallo** [angular/material2#13094]
#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
Chrome
![image](https://user-images.githubusercontent.com/10036108/45430274-55320280-b6a5-11e8-8732-c6668590a2e0.png)

#### What is the current behavior?
Firefox
![image](https://user-images.githubusercontent.com/10036108/45430370-83174700-b6a5-11e8-8c4e-46c214595cc0.png)

Edge
![image](https://user-images.githubusercontent.com/10036108/45430430-99250780-b6a5-11e8-9e6e-837a6d560b2b.png)

#### What are the steps to reproduce?
```html
<mat-form-field>
    <mat-label>Label</mat-label>

    <input matInput>

    <button mat-icon-button matSuffix><mat-icon>cancel</mat-icon></button>
</mat-form-field>
```

#### What is the use-case or motivation for changing an existing behavior?
Keep ripple effect on click of icon-button.

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Angular 6.1.2
Angular Material 6.4.7
TS 2.9.x
Firefox, Edge

#### Is there anything else we should know?
I will switch to mat-icon for now, for which everything works fine.
On Material Design guidelines it's not clear if a suffix/prefix element could/should/must have or not a ripple effect, was just trying to be consistent with all others clickable icons.

**Comment by manklu** [comment]
@IlCallo I have tried your snippet and it looks the same in all three browsers

https://stackblitz.com/edit/angular-material2-issue-6qx8gu

**Comment by IlCallo** [comment]
You're right, I forgot to mention it happens with `fill` appearance (I thought it was obvious from the screenshots), I updated the title and the code and linked a modified version of your stackblitz :)

**Comment by zbream** [comment]
Still running into this. Notably, it affects the use of the datepicker toggle as a suffix.

**Comment by crisbeto** [comment]
We are closing this issue, because we've determined, using a semi-automated process, that it is no longer relevant, has low engagement, or it does not have enough information to proceed. Feel free to comment if you think the issue should be reopened, or open a new issue with the latest version of our code instead.
